src/stories/theme-styling.css

```diff
@@ -0,0 +1,204 @@
+  --ck-color-button-on-active-background: var(--ck-color-button-default-active-background);
+  --ck-color-button-on-active-shadow: transparent; /* TODO NEEDS WORK */
+  --ck-color-button-on-disabled-background: var(--ck-color-button-default-disabled-background);
+
+  /* DROPDOWNS */
+  --ck-dropdown-arrow-size: var(--ck-spacing-extra-large);
+}
+
+/* BUTTONS */
+.ck.ck-button:not(.ck-disabled):hover .ck-button__icon {
+  color: #49545c; /* GREY-700 */
+}
+
+.ck.ck-button:not(.ck-disabled) {
+  cursor: pointer;
+}
+
+/* GLOBAL EDITOR */
+.ck.ck-editor__editable_inline {
+  padding: 0 var(--ck-spacing-large);
+}
+
+/* PANELS */
+.ck.ck-balloon-panel {
+  box-shadow: var(--ck-drop-shadow);
+}
+
+/* TOOLBAR */
+.ck.ck-toolbar__separator {
+  margin-right: var(--ck-spacing-standard) !important;
+  margin-left: var(--ck-spacing-standard) !important;
+}
+
+/* HORIZONTAL LINE */
+.ck-content hr {
+  margin: var(--ck-spacing-standard) 0;
+}
+
+/* Blockquote */
+.ck-content blockquote {
+  overflow: hidden;
+  padding: 0 0 0 calc(var(--ck-spacing-standard) * 4);
+  font-style: inherit;
+  border-left: 2px solid #c2c8cc; /* GREY-400 */
+}
+
+.ck-content[dir='rtl'] blockquote {
+  padding: 0 calc(var(--ck-spacing-standard) * 4) 0 0;
+  border-left: 0;
+  border-right: 2px solid #c2c8cc; /* GREY-400 */
+}
+
+/* TYPOGRAPHY */
+.ck-content h1 {
+  line-height: 32px;
+  font-size: 26px; /* XXL */
+  font-weight: 600;
+  margin-block-start: 8px;
+  margin-block-end: 8px;
+}
+
+.ck-content h2 {
+  line-height: 28px;
+  font-size: 22px; /* XL */
+  font-weight: 600;
+  margin-block-start: 8px;
+  margin-block-end: 8px;
+}
+
+.ck-content h3 {
+  line-height: 24px;
+  font-size: 18px; /* LG */
+  font-weight: 600;
+  margin-block-start: 8px;
+  margin-block-end: 8px;
+}
+
+.ck-content h4 {
+  line-height: 20px;
+  font-size: 14px; /* MD */
+  font-weight: 600;
+  margin-block-start: 8px;
+  margin-block-end: 8px;
+}
+
+.ck-content p {
+  margin-bottom: 20px;
+}
+
+.ck-content strong {
+  font-weight: 600;
+}
+
+.ck-content u {
+  text-decoration: underline;
+}
+
+/* BLOCKQUOTE */
+.ck-content blockquote {
+  margin-block-start: 16px;
+  margin-block-end: 16px;
+}
+
+.ck-content blockquote p:last-child {
+  margin-bottom: 0;
+}
+
+.ck-content pre {
+  margin-block-start: 16px;
+  margin-block-end: 16px;
+}
+
+/* LISTS */
+.ck-content ul {
+  margin: 20px 0px 20px 24px;
+  padding: 0;
+  list-style-position: outside;
+  list-style-type: disc;
+}
+
+.ck-content ol {
+  margin: 0px 0px 0px 24px;
+  padding: 0;
+  list-style-position: outside;
+  list-style-type: decimal;
+}
+
+/* CODE */
+.ck-content code {
+  font-family: SFMono-Regular, Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+  font-size: calc(1em - 1px);
+  direction: ltr;
+  border-radius: 2px;
+  padding: 1.5px;
+  background-color: rgb(233, 235, 237); /* GREY-200 */
+  color: rgb(73, 84, 92);
+}
+
+.ck.ck-editor__editable .ck-code_selected {
+  background-color: #d8dcde; /* GREY-300 */
```

Comment:
@ginnywood @m-lai why is it desirable to darken the background color on `Code` selection? This doesn't seem consistent with the preformatted text block.

Reply:
That is a good point. Let’s remove it.

Reply:
Since `ck-code_selected` is a part of the `code` plugin we still have to override the original value. I've updated this to match the default `grey-200` state.